featurebyte/worker/util/batch_feature_creator.py

```diff
@@ -351,9 +362,20 @@ async def batch_feature_create(
         )
         await ranged_progress_update(0, "Started saving features")
 
-        output_feature_ids = []
+        output_feature_ids: List[PydanticObjectId] = []
         inconsistent_feature_names = []
         for i, feature_item in enumerate(payload.features):
+            if (len(output_feature_ids) + 1) % graph_clear_frequency == 0:
```

Comment:
Does `output_feature_ids` include features that were already saved and skipped?

Reply:
Good catch. Could use another counter to account for the non-skipped feature. Will update this part.